unit-tests/shared/src/test/scala/javalib/util/stream/StreamTest.scala

```diff
@@ -46,6 +46,13 @@ class StreamTest {
     assertFalse(it.hasNext())
   }
 
+  @Test def streamForeachWorks(): Unit = {
+    val s = Stream.of(1, 2, 3)
+    var sum = 0
+    s.forEach(sum += _)
```

Comment:
This line seems to cause issues for Scala 2.11
```scala
[error] D:\a\scala-native\scala-native\unit-tests\shared\src\test\scala\javalib\util\stream\StreamTest.scala:52:22: missing parameter type for expanded function ((x$1) => sum.<$plus$eq: error>(x$1))
[error]     s.forEach(sum += _)
[error]                      ^
[error] one error found
[error] (testsJVM2_11 / Test / compileIncremental) Compilation failed
```
https://github.com/scala-native/scala-native/runs/7479078735?check_suite_focus=true#step%3A5%3A69=

Reply:
Thanks :)